routing_model/src/flows_dst.rs

```diff
@@ -343,6 +360,23 @@ impl CheckAndProcessElderChange {
         self.0.action.schedule_event(LocalEvent::TimeoutCheckElder);
         self.clone()
     }
+
+    fn vote_parsec_neighbour_merge(&self) -> Self {
+        self.0
+            .action
+            .vote_parsec(ParsecVote::NeighbourMerge(NeighbourMergeInfo));
+        self.clone()
+    }
+
+    fn store_merge_infos(&self, info: NeighbourMergeInfo) -> Self {
+        let mut state = self.clone();
+        state.mut_routine_state().neighbour_merge_info = Some(info);
```

Comment:
No, this is a function in the graph.
What this mean is that it needs to be implemented by the Action component.
Only local variable that appear in the graph as SCREAMING_CAMEL_CASE should be stored locally, or any local state you find yourself needing.

Reply:
Ah yes good point!